src/index.js

```diff
@@ -77,153 +81,182 @@ export default class OneSphere {
 
   // users
 
-  getUsers() {
-    return fetcher(`${this.host}/rest/users`, { headers: this.headers });
+  getUsers(args) {
+    const params = args ? getUrlParams(args) : '';
```

Comment:
Lets DRY this up a bit, we shouldn't be using the same ternary over and over. How about a `handleArgs` method that has the ternary? This way if we every want to handle defining the `params` differently we only need to change one block. Let me know if that doesn't make sense.